src/ZenHubIssueClient.cs

```diff
@@ -48,6 +48,22 @@ public async Task<Response<Models.IssueEvent[]>> GetEventsAsync(CancellationToke
         /// </summary>
         /// <param name="estimate">The value of the estimate</param>
         public async Task<Response> SetEstimateAsync(int estimate, CancellationToken cancellationToken = default)
+        {
+            return await SetEstimateInternalAsync(estimate, cancellationToken)
+                .ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Set the estimate on the issue
+        /// </summary>
+        /// <param name="estimate">The value of the estimate</param>
+        public async Task<Response> SetEstimateAsync(double estimate, CancellationToken cancellationToken = default)
+        {
+            return await SetEstimateInternalAsync(estimate, cancellationToken)
+                .ConfigureAwait(false);
+        }
+
+        private async Task<Response> SetEstimateInternalAsync<T>(T estimate, CancellationToken cancellationToken = default)
```

Comment:
Interesting - I don't see where this is called. Binding doesn't "just work" does it? Or are you not calling it yet?

Reply:
Line 52 and Line 62